example-scenegraph-sdk/source/components/helloworld.brs

```diff
@@ -16,9 +16,15 @@ sub init()
     identityApiRequest.userIdentities = {}
     identityApiRequest.userIdentities[mpConstants.IDENTITY_TYPE.OTHER] = "foo"
     m.mparticle.identity.modify(identityApiRequest)
+    dentityApiRequest = {}
```

Comment:
Compilation issue - maybe now is the time to add bslint to the repo, you could just add onto the new action we added the other day: https://github.com/rokucommunity/bslint

Reply:
yea ill check it out. I certainly can't keep my cat off the keyboard to avoid this form happening
<img width="1449" alt="Screen Shot 2020-10-16 at 11 46 21 AM" src="https://user-images.githubusercontent.com/33703490/96297800-70c41180-0fbf-11eb-9ce4-0d81d77e80c9.png">